tox.ini

```diff
@@ -24,14 +24,16 @@ deps =
     sqlalchemy13: sqlalchemy>=1.3,<1.4
     sqlalchemy14: sqlalchemy>=1.4,<1.5
 
-[testenv:django{111,20,21,22}-only]
+[testenv:django{111,20,21,22,32,40}-only]
 commands = {toxinidir}/tests/test_django/test.sh --no-input {posargs}
 extras = development
 deps =
     django111: django>=1.11,<2.0
     django20: django>=2.0,<2.1
     django21: django>=2.1,<2.2
     django22: django>=2.2<2.3
+    django32: django>=3.2<3.3
+    django32: django>=4.0<4.1
```

Comment:
I think you mean to make this line `django40`

Reply:
Nice catch!